src/clickhouse/handleSinkRequest.ts

```diff
@@ -86,59 +91,73 @@ function batchSizeLimitReached() {
   return bufferedItems >= config.maxBufferSize;
 }
 
-function handleNoEntityChange(metadata: { clock: Clock; manifest: Manifest; cursor: string }) {
+function handleEntityChanges(entityChanges: EntityChange[], metadata: Metadata) {
+  logger.info(`handleSinkRequest | entityChanges=${entityChanges.length}`);
+  for (const change of entityChanges) {
+    const values = getValuesInEntityChange(change);
+    handleChange(change.entity, values, change.operation, { ...metadata, id: change.id });
+  }
+}
+
+function handleDatabaseChanges(tableChanges: TableChange[], metadata: Metadata) {
+  logger.info(`handleSinkRequest | tableChanges=${tableChanges.length}`);
+  for (const change of tableChanges) {
+    const values = getValuesInTableChange(change);
+    handleChange(change.table, values, change.operation, { ...metadata, id: "" });
```

Comment:
Should the `id` be empty or the primary key in `db_out`?

Reply:
Oh that's a good question... the primary key in `db_out` is `pk` or `compositePk`